packages/js/src/ui/components/Renderer.tsx

```diff
@@ -15,7 +15,7 @@ import { UnreadCountProvider } from '../context/UnreadCountContext';
 import { Bell, Root } from './elements';
 import { Inbox } from './Inbox';
 
-const NovuComponents = {
+export const novuComponents = {
   Inbox,
   Bell,
```

Comment:
We need to export Settings and Notification list components too

Reply:
We will :)

Reply:
I'd also want to remind you that we need to export a Notifications components that also accepts `renderNotification` prop.

Reply:
Yeap, this is PR is like an MVP to show how we will be doing this back and forth.